assets/css/_presets.scss

```diff
@@ -6,7 +6,7 @@
     li {
       display: flex;
       justify-content: space-between;
-      align-items: flex-end;
+      align-items: center;
```

Comment:
question (non-blocking): Intentional change to align-items center? Seems like it could affect more than the close button

Reply:
Correct - previously the preset name and the close button were about the same height and `flex-end` gave us the alignment we wanted, but with the larger button and slightly larger font it looked pretty weird.